beancount/parser/grammar.y

```diff
@@ -409,22 +410,28 @@ price_annotation : incomplete_amount
                      $$ = $1;
                  }
 
-posting : INDENT optflag ACCOUNT incomplete_amount cost_spec eol
+account : ACCOUNT
```

Comment:
There are a lot of accounts... this is adding another call for each IIUC.
Did you benchmark the difference?

Reply:
Combined with the changes in the other PR, the call is moved from the lexer to the grammar, thus there should be no performance penalty. I haven't benchmarked this change alone. With both PRs applied, parsing gets 50% faster IIRC.